samples/common/buffers.h

```diff
@@ -393,6 +393,24 @@ class BufferManager
         memcpyBuffers(false, true, false);
     }
 
+    //!
+    //! \brief Copy the selected contents of output device buffers by bidingIdx to output host buffers synchronously.
+    //!
+    void copyOutputToHost(const std::vector<int>& bindingIndices) 
```

Comment:
Thanks for the contribution!

To follow our coding convention, it would be great if you can make the following modifications:

- Use east-const style (i.e. `const std::vector<int>&` -> `std::vector<int> const&`) for all occurrences of `const`.
- Use `int32_t` instead of `int`.
- Use 4-whitespace indentation instead of 2-whitespace indentation.

We appreciate it!